### PR TITLE
Update Plugin.nuspec

### PR DESCRIPTION
**Pull request by mclillill**

### CODE REVIEW

File: nuget/Plugin.nuspec [hunk 1]
```diff
@@ -20,7 +20,7 @@
            <dependency id="NETStandard.Library" version="1.6.1" />
          </group>
         <group targetFramework="MonoAndroid80">
-            <dependency id="Xamarin.Firebase.Messaging" version="[60.1142.0]" />
+            <dependency id="Xamarin.Firebase.Messaging" version="[60.1142.1]" />
```
Comment (dationguyen) [inline nuget/Plugin.nuspec]: can we use flexible version number instead? 
the fixed version number may conflict with other packages if the app is using higher firebase version.

Comment (mclillill) [reply nuget/Plugin.nuspec]: Yes. I think 60.0 would be fine. (Without the brackets.)
Syntax here:
https://docs.microsoft.com/de-de/nuget/concepts/package-versioning#version-ranges-and-wildcards

Comment (dationguyen) [reply nuget/Plugin.nuspec]: do we have any plan for this? this nuget is out of date for a year

**Comment by dationguyen** [comment]
@rdelrosario can re remove the fixed version number? (remove bracket [] )

**Pull request merged**